client/src/less/progress.less

```diff
@@ -0,0 +1,51 @@
+.container {
```

Comment:
You might be interested in reviewing [some of the powerful features LESS gives you](http://lesscss.org/features/#variables-feature), but keep in mind that LESS is backwards compatible, so all your CSS knowledge applies. Now is the time for experimentation!